test_util/dcos_api_session.py

```diff
@@ -278,6 +318,7 @@ def _wait_for_metronome(self):
     def wait_for_dcos(self):
         self._wait_for_adminrouter_up()
         self._authenticate_default_user()
+        self.set_node_lists_if_unset()
```

Comment:
I flirted with making this public, but that no longer has a use case; will make it private as well.